lnwire/neighbor_ack_test.go

```diff
@@ -0,0 +1,44 @@
+// Copyright (c) 2016 Bitfury Group Limited
+// Distributed under the MIT software license, see the accompanying
+// file LICENSE or http://www.opensource.org/licenses/mit-license.php
+
+package lnwire
+
+import (
+       "bytes"
+       "testing"
+
+       "github.com/roasbeef/btcd/wire"
+)
+
+func TestNeighborAckMessageEncodeDecode(t *testing.T) {
+       b := new(bytes.Buffer)
+       msg1 := NeighborAckMessage{}
+       err := msg1.Encode(b, 0)
+       if err != nil {
+               t.Fatalf("Can't encode message ", err)
+       }
+       msg2 := new(NeighborAckMessage)
+       err = msg2.Decode(b, 0)
+       if err != nil {
+               t.Fatalf("Can't decode message ", err)
+       }
+}
+
+func TestNeighborAckMessageReadWrite(t *testing.T){
```

Comment:
There was a bug with messages of size zero. This test ensures that writing/reading (with header) given message goes without errors.